$NetBSD: patch-ah,v 1.4 2011/12/24 17:07:07 dholland Exp $

- use standard headers
- don't declare own errno
- don't rely on "unix" being defined, as it isn't

--- src/fileio.c.orig	1995-06-23 10:25:18.000000000 +0000
+++ src/fileio.c
@@ -117,10 +117,7 @@ the Free Software Foundation, 675 Mass A
 
 #include <errno.h>
+#include <stdlib.h>
 
-#ifndef vax11c
-extern int errno;
-#endif
-
 extern char *strerror ();
 
 #ifdef APOLLO
@@ -2604,8 +2601,6 @@ The value is an integer.")
   return value;
 }
 
-#ifdef unix
-
 DEFUN ("unix-sync", Funix_sync, Sunix_sync, 0, 0, "",
   "Tell Unix to finish all pending disk updates.")
   ()
@@ -2614,8 +2609,6 @@ DEFUN ("unix-sync", Funix_sync, Sunix_sy
   return Qnil;
 }
 
-#endif /* unix */
-
 DEFUN ("file-newer-than-file-p", Ffile_newer_than_file_p, Sfile_newer_than_file_p, 2, 2, 0,
   "Return t if file FILE1 is newer than file FILE2.\n\
 If FILE1 does not exist, the answer is nil;\n\
@@ -4507,7 +4500,5 @@ This applies only to the operation `inhi
   defsubr (&Sread_file_name_internal);
   defsubr (&Sread_file_name);
 
-#ifdef unix
   defsubr (&Sunix_sync);
-#endif
 }
